apis/ec2-2016-11-15.normal.json

```diff
@@ -12006,7 +12006,7 @@
         },
         "ImageIds": {
           "shape": "ImageIdStringList",
-          "documentation": "<p>The image IDs.</p> <p>Default: Describes all images available to you.</p>",
+          "documentation": "<p>The image IDs. Example: <code>{Name: 'image-id', Values: ['ami-xxxxxxxxxxx']}</code></p> <p>Default: Describes all images available to you.</p>",
```

Comment:
Unfortunately, any changes to this or any of the APIs files will be overwritten with the next release.

These API models come from the individual services.   They are consumed by this SDK to generate the clients and documentations.  As such, we take this part of the PR.

Reply:
Then how can we make sure that we make the API models to generate these? Is that in the same repository?

Reply:
The API Documentation is managed by the service teams.  The models are shared by all of the SDKs. They won't be able to include this kind of specific example.  It'll need to be removed from this PR before the changes to CONTRIBUTING.md can be accepted.

Reply:
Done, have updated the PR to remove this entry.